File: practice/09/c++/9/9/9.cpp
#include <iostream>
#include <sstream>

const int MINUTES_IN_HOUR = 60;

int main() {
    setlocale(LC_ALL, "Russian");
    std::cout << "Введите 2 момента времени в формате \"ЧЧ:ММ\", каждое с новой строки." << std::endl;
    int h1, m1, h2, m2;
    char temp;
    std::cin >> h1 >> temp >> m1 >> h2 >> temp >> m2;
    m1 += h1 * MINUTES_IN_HOUR;
    m2 += h2 * MINUTES_IN_HOUR;

    if (std::abs(m1 - m2) <= 15) {
        std::cout << "Встреча состоится!" << std::endl;
    }
    else {
        std::cout << "Встреча не состоится!" << std::endl;
    }
}

File: practice/09/Python/9.py
MINUTES_IN_HOUR = 60

print('Введите 2 времени в формате \"ЧЧ:ММ\" каждое с новой строки')

h1, m1 = map(int, input().split(sep=':'))
h2, m2 = map(int, input().split(sep=':'))

m1 = MINUTES_IN_HOUR * h1 + m1
m2 = MINUTES_IN_HOUR * h1 + m2

if abs(m1 - m2) > 15:
    print('Встреча не состоится')
else:
    print('Встреча состоится')